docs/components/periodic-updates.rst

```diff
@@ -0,0 +1,54 @@
+DKAN Periodic Updates
+=====================
+
+The DKAN Periodic Updates module provides the user the ability to execute automated updates on existing resources in a daily, weekly or monthly basis. It also provides a UI in which you can check at the status of the updates.
+The module requires a CSV file to execute the updates, this file is known as "the manifest", when you upload it and enable the updates, this is what happens:
+
+- The manifest is evaluated on every cron run to determine what resources needs to be updated (based on the frequency defined in the manifest and the "last update" date).
+- For each resource that needs an update, the system will:
+  - Download the file URL specified in the manifest (or link it, according to which file field is used on the corresponding resource).
+  - Update the resource node to use the new file.
+  - If the file is a CSV or TSV, then the resource is imported into the datastore.
+
+Periodic Updates Settings
+-------------------------
+
+You can find the periodic updates settings under DKAN --> Periodic Updates --> Settings (`/admin/dkan/periodic-updates`).
+The settings for this module includes:
+
+  - Periodic updates state
+  - Manifest for periodic updates
+
+1- Periodic updates state
+*************************
+
+To turn periodic updates on you need to be in the Periodic Updates page, once there you should mark the option "Enable periodic updates" and click on "Save settings". You'll also need to upload a valid manifest in the section "Manifest for periodic updates", otherwise the updates cannot be executed.
```

Comment:
~To turn periodic updates on you need to be in the Periodic Updates page, once there you should mark the option "Enable periodic updates" and click on "Save settings".~

Enable periodic updates by checking the "Enable periodic updates" box on the Periodic Updates page, and click on "Save settings".